app/controllers/application_controller.rb

```diff
@@ -1,5 +1,19 @@
 class ApplicationController < ActionController::Base
-def current_user
-    @current_user = User.first
+  #before_action :authenticate_user!
```

Comment:
- Good job done so far, but I can still access your website without registration this is because you have commented out this line of code, I suggest that you authenticate a user before they can access any of your controller actions. Please fix this 🙏

Reply:
Thanks a lot !